lib/constants/constants.dart

```diff
@@ -12,3 +12,4 @@ Color iconColor = const Color.fromARGB(255, 173, 173, 173);
 Color black = Colors.black;
 Color white = Colors.white;
 Color drawerHeaderColor = const Color.fromARGB(255, 129, 128, 128);
+TextStyle buttonTextStyle = TextStyle(color: white, fontSize: 20);
```

Comment:
It looks like you saved this in two locations: (1) here in the constants and (2) as a class-level variable. Please remove it from here since all the usages are limited to that one file?